brainreg_segment/regions/layers.py

```diff
@@ -74,6 +74,13 @@ def add_existing_label_layers(
     return label_layer
 
 
+def add_region_from_existing_layer(selected_layer, label_layers):
+    """
+    Adds an existing label layer (e.g. from another plugin) to the list
+    """
+    label_layers.append(selected_layer)
```

Comment:
```suggestion
def add_region_from_existing_layer(new_layer, label_layers):
    """
    Adds an existing label layer (e.g. from another plugin) to the list
    """
    label_layers.append(new_layer)
```
since in this context it's not neccesarily a 'selected' layer being added.